CHANGELOG.md

```diff
@@ -2,5 +2,8 @@
 
 ## Current Main Branch
 
+## 1.0.1 - Jul 22, 2022
+- [#134](https://github.com/MetaMask/metamask-mobile/pull/134): [FIX] - remove categories filter for iOS
```

Comment:
Noticed the PR url goes to `metamask/metamask-mobile` instead of `metamask/dapps`. Should we change that?